src/renderer/components/Onboarding/LangSwitcher.js

```diff
@@ -38,24 +39,24 @@ const styleFn = theme => ({
   },
   valueContainer: (provided, state) => ({
     ...provided,
-    paddingLeft: 14,
-    paddingRight: 0,
+    paddingLeft: rtl ? 0 : 14,
+    paddingRight: rtl ? 14 : 0,
```

Comment:
Instead of relying on `rtl`, can't you use `paddingInlineStart` and `paddingInlineEnd` ? Or `paddingBlockStart`and `paddingBlockEnd` depending on your elements.